src/App.js

```diff
@@ -1,18 +1,34 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route } from 'react-router-dom'
+import './css/index.css';
```

Comment:
please remove it has already been added in index.js...

Reply:
I have to reference it on every component since they are not in the same package

Reply:
index.css is supposed to propagate, you already imported it in index.js, and it is the same index.css file.

Reply:
I solved.